src/backend/api/meals.js

```diff
@@ -2,13 +2,28 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
-router.get("/", async (request, response) => {
-  try {
+//router.get("/", async (request, response) => {
+  //try {
     // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
-  } catch (error) {
-    throw error;
+   // const titles = await knex("meals").select("title");
+    //response.json(titles);
+  //} catch (error) {
+    //throw error;
+  //}
+//});
+
+router.get("/", async (request, response) => {
```

Comment:
I highly recommend getting into a habit of formatting the code before committing. You can set VS code to do it automatically as well upon save :)